vendor/github.com/evergreen-ci/bond/.golangci.yml

```diff
@@ -0,0 +1,11 @@
+---
+linters:
+  disable-all: true
+  enable:
+    - errcheck
+    - goimports
+    - govet
+    - ineffassign
+    - interfacer
```

Comment:
Can we remove interfacer? I'm trying to get rid of it from all the repos because all it does is suggest when an interface can be used instead of a concrete type, which is often undesirable.

Reply:
wouldn't that have to be a change in bond?

Reply:
Hmm yeah I guess. I'll do it in a later ticket then since I have to set up CI testing for bond anyways.